src/NServiceBus.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -201,26 +201,33 @@ void Connection_ConnectionShutdown(object sender, ShutdownEventArgs e)
             }
         }
 
+        class MessageState
+        {
+            public IModel Channel { get; set; }
+
+            public ulong DeliveryTag { get; set; }
+        }
+
         Task AcknowledgeMessage(IModel channel, ulong deliveryTag)
         {
-            var task = new Task(() =>
+            return Task.Factory.StartNew(messageState =>
             {
-                channel.BasicAck(deliveryTag, false);
-            });
+                var s = (MessageState)messageState;
```

Comment:
Silly nitpick, but as a way of avoiding `s`, I'd probably prefer `state` for the lambda param and `messageState` for the local.